providers/pool.rb

```diff
@@ -13,7 +13,17 @@ def whyrun_supported?
 
 action :create do
   if @current_resource.exists
-    Chef::Log.info "#{@new_resource} already exists - nothing to do."
+    if @current_resource.pg_num == @new_resource.pg_num
```

Comment:
I felt it was redundant - pg_num should never differ from pgp_num except while changing values.  We could test for it just in case something fails though.